### DISCUSSION TITLE
Need to lower uart baud rate for Windows?

### DISCUSSION THREAD

**Discussion by grtwje** [rust-embedded/rust-raspberrypi-OS-tutorials#103] [Tutorial Questions]
I working on 06_drivers_gpio_uart. I have the Adafruit serial cable. miniterm sits waiting but never connects.  My environment is WSL. I installed the adafruit Windows driver, and it shows up on COM3.  I see in the uart code that the baud rate is set to 921,600.  Since Windows only allows COM3 to go up to 128,000, I wonder if this is my problem? I'm new to this and thought I should see if I'm on the right track before wasting too much time.

**Comment by grtwje** [comment]
While the baud rate might be a problem, there's a bigger WSL 2 issue blocking access, [4322](https://github.com/microsoft/WSL/issues/4322).

**Reply by umaYnit** [reply]
I use it on Win10, 921_600 is ok.
